modals-notifications/es5/src/js/notification.js

```diff
@@ -0,0 +1,55 @@
+function Notification(config, duration) {
+	Parent.prototype.constructor.apply(this, arguments);
+
+	this.parent = document.querySelector('.notificationsJS');
+	this.duration = duration;
+	this.html = '' +
+		'<div class="notification notificationJS">' +
+			'<div class="notification__icon">' +
+				'<img src="img/' + this.config.icon + '.svg" alt="icon">' +
+			'</div>' +
+			'<div class="notification__content">' +
+				'<span class="notification__title notification__title_' + this.config.type + '">' + this.config.title + '</span>' +
+				'<p class="notification__desc">' + this.config.desc + '</p>' +
+			'</div>' +
+			'<span class="notification__close notificationCloseJS"></span>' +
+		'</div>';
+
+}
+
+extend(Notification, Parent);
+
+Notification.prototype = {
+	show: function () {
+		Notification.superclass.show.apply(this);
+
+		var notifications = this.parent.querySelectorAll('.notificationJS');
+
+		notifications[0].setAttribute('data-timer-id',this.timerHideNotification(this.duration));
```

Comment:
why notifications[0] ? What if you need [1] ? or [2] ?

Reply:
i'll explain.
when you add new notification to the parent wrap it will be first in nodeList and it's getting data-id from function 'timerHideNotification'

in a nutsshell every new notification gets data-id from function,
'timerHideNotification' return id and runs your script

ps sorry for my english) i thought that i will try